### PR TITLE
README cleanup

### PR DESCRIPTION
**Pull request by marcelamelara**
Makes the README more contributor-friendly, and moves a lot of the old text to files in the new docs/ directory. 

Signed-off-by: Marcela Melara <marcela.melara@intel.com>

cc/ @pxp928 @adityasaky

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Maintainers
+
+### Active Maintainers
+| Name                       | Github          |
+|----------------------------|-----------------|
+| Tom Hennen (Google)        | [@TomHennen](https://github.com/TomHennen) |
```
Comment (TomHennen) [inline MAINTAINERS.md:6]: nit: Could this be alphabetized?  I wouldn't want folks to read anything into the order here.

**Review by TomHennen — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (marcelamelara) [reply MAINTAINERS.md:6]: It's currently in family-name alphabetical order, do you mean it should alphabetized by first name?

Comment (TomHennen) [reply MAINTAINERS.md:6]: Oh! I see. I was naively just looking at first name.  I think I was surprised to see myself first :)

Comment (marcelamelara) [reply MAINTAINERS.md:6]: Ah I see. First name is the more obvious order, so I'll change it :)

File: README.md [hunk 2]
```diff
@@ -1,299 +1,75 @@
...
-\[1]: The `expected_command` is only a warning, and `inspections` require
-running external commands which is infeasible in many situations.
-
-\[2]: That said, we strongly recommend against keying a database purely by
-content hash. The reason is that such databases quickly run into scaling issues,
-as explained in
-[Building Secure and Reliable Systems](https://static.googleusercontent.com/media/landing.google.com/en//sre/static/pdf/Building_Secure_and_Reliable_Systems.pdf#page=364),
-Chapter 14, page 328, "Ensure Unambiguous Provenance." Instead, we recommend
-keying primarily by resource name, in addition to content hash.
+# In-toto Attestation Framework
```
Comment (joshuagl) [inline README.md:1]: Nit: AIUI in-toto is always lowercase

```suggestion
# in-toto Attestation Framework
```

File: README.md [hunk 3]
```diff
@@ -1,299 +1,75 @@
...
+
+<!-- msm: This is real clunky -->
+The in-toto Attestation Framework is a framework for validating claims about
+any aspect of how a piece of software is produced, allowing anyone
+to verify the origins of software and establish trust in its supply chain.
```
Comment (joshuagl) [inline README.md:6]: Are we providing a framework for validating claims, or a framework for making claims which can be validated? The SLSA docs on the [attestation model](https://slsa.dev/attestation-model) say that:

> A software attestation is an authenticated statement (metadata) about a software artifact or collection of software artifacts.
>
> The primary intended use case is to feed into automated policy engines, such as [in-toto](https://in-toto.io/) and [Binary Authorization](https://cloud.google.com/binary-authorization).

**Review by joshuagl — COMMENTED** [review]
This is neat set of improvements, thanks @marcelamelara. I think we should fix up the linter issues (and perhaps wrap lines consistently at 80chars), then it's good to land and iterate on in-tree.

**Review by pxp928 — APPROVED** [review]
LGTM thanks @marcelamelara

File: README.md (hunk 3 above)
Comment (marcelamelara) [reply README.md:6]: I wanted to up-level the description of the framework a bit more, but good point. Right now our focus is on specifying/generating these verifiable claims. I'll change the wording on this, thanks!

File: README.md [hunk 4]
```diff
@@ -1,299 +1,83 @@
-# In-toto Attestations
+# in-toto Attestation Framework
 
-This repository defines the in-toto **attestation** format, which represents
-authenticated metadata about a set of software artifacts. Attestations are
-intended for consumption by automated policy engines, such as [in-toto] and
-[Binary Authorization].
+<!-- msm: This is real clunky -->
+The in-toto Attestation Framework is a framework for generating verifiable
```
Comment (adityasaky) [inline README.md:4]: The in-toto Attestation Framework describes mechanisms for generating verifiable claims about how a piece of software is produced. Any developer or consumer of some software can validate its origins and establish trust in its software supply chain.


Not sure this is a whole lot better but:
1. I didn't want to use framework twice right at the start
2. I think splitting it up into two sentences makes it easier to parse

Comment (adityasaky) [reply README.md:4]: Also, best argument to not have "framework" in the name? :smile:

File: MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Maintainers
+
+## Active Maintainers
```
Comment (adityasaky) [inline MAINTAINERS.md:3]:
```suggestion
```

We don't need a subheading, IMO

File: MAINTAINERS.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+| Name                       | Github          |
```
Comment (adityasaky) [inline MAINTAINERS.md:5]:
```suggestion
| Name                       | GitHub          |
```

nit

**Review by adityasaky — APPROVED** [review]
LGTM apart from a couple of minor comments.

File: README.md (hunk 4 above)
Comment (marcelamelara) [reply README.md:4]: Yeah... this is about the only time when having "framework" in the name is inconvenient... But agreed on breaking up that sentence!

File: MAINTAINERS.md (hunk 5 above)
Comment (marcelamelara) [reply MAINTAINERS.md:3]: done.

File: MAINTAINERS.md (hunk 6 above)
Comment (marcelamelara) [reply MAINTAINERS.md:5]: done

File: README.md (hunk 4 above)
Comment (marcelamelara) [reply README.md]: Please let me know if you like the updated wording better!

Comment (adityasaky) [reply README.md]: Loads better!

**Pull request merged**